README.rst

```diff
@@ -138,6 +138,13 @@ To build DALI with LMDB support:
    cmake -DBUILD_LMDB=ON ..
    make -j"$(nproc)"
 
+To build DALI using clang:
+
+.. code-block:: bash
+   
+   cmake -DCMAKE_CXX_COMPILER=clang++ -DCMAKE_C_COMPILER=clang  ..
```

Comment:
Please add note that is it experimental and we are not testing that thoroughly and we recommend to use gcc for the productions builds.

Reply:
Done